listenbrainz_spark/recommendations/recommend.py

```diff
@@ -367,7 +391,8 @@ def main(recommendation_top_artist_limit=None, recommendation_similar_artist_lim
                                   recommendation_top_artist_limit,
                                   recommendation_similar_artist_limit)
 
-    messages = get_recommendations_for_all(params, users)
+    ti = time.monotonic()
+    messages = get_recommendations_for_all(params, users, ti)
```

Comment:
Do we actually need to pass the ti from here? Can't we just  calculate it in the get_recommendations function? If we can, we should do that, because the extra parameter makes the function harder to use.